### PR TITLE
Convert README to Markdown

### PR DESCRIPTION
**Pull request by KapJI**
This may be personal preference but I think Markdown won RST as a common standard. I think RST is too powerful for such a simple use case.
Also I want to add `CONTRIBUTING.md` with intstructions how to use Poetry, it's better to have everything in the same format.
- Add badges for CI status and PyPI package.

Screenshot 2021-04-01 at 18 56 33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# GPSOAuth
```
Comment (simon-weber) [inline README.md:1]: I think I'd rather leave it as is for consistency's sake with other references (ports, issues, etc).

File: pyproject.toml [hunk 2]
```diff
@@ -16,7 +16,7 @@ classifiers = [
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
 ]
-readme = "README.rst"
+readme = "README.md"
```
Comment (simon-weber) [inline pyproject.toml:19]: this is left-over from back when pypi didn't support markdown. It looks like they changed that at some point; so long as it shows up fine markdown is cool by me.

File: README.md (hunk 1 above)
Comment (KapJI) [reply README.md]: Sure, reverted this.

File: pyproject.toml (hunk 2 above)
Comment (KapJI) [reply pyproject.toml:19]: It works just fine now!
For example: [`glocaltokens`](https://github.com/leikoilja/glocaltokens), rendered the same on [PyPI](https://pypi.org/project/glocaltokens/).

**Comment by simon-weber** [comment]
thanks! if you get a chance it'd be cool to try out the new readme on Test PyPI; otherwise I'll get around to it eventually.

**Comment by KapJI** [comment]
Here it is: https://test.pypi.org/project/gpsoauth-test/

**Comment by simon-weber** [comment]
awesome; thanks!

**Pull request merged**